pkg/volume/csi/csi_block.go

```diff
@@ -67,10 +68,10 @@ func (m *csiBlockMapper) getStagingPath() string {
 }
 
 // getPublishPath returns a publish path for a file (on the node) that should be used on NodePublishVolume/NodeUnpublishVolume
-// Example: plugins/kubernetes.io/csi/volumeDevices/publish/{pvname}
+// Example: plugins/kubernetes.io/csi/volumeDevices/publish/{pvname}/{podUid}
 func (m *csiBlockMapper) getPublishPath() string {
 	sanitizedSpecVolID := utilstrings.EscapeQualifiedName(m.specName)
-	return filepath.Join(m.plugin.host.GetVolumeDevicePluginDir(CSIPluginName), "publish", sanitizedSpecVolID)
```

Comment:
shouldn't sanitizedSpecVolID be m.volumeID, not come from the PV name?

Reply:
the spec says publish/stage should be idempotent per volumeID & m.volumeID is taken from PV volumeHandle

Reply:
Changed to use volumeID.